docs/.vuepress/enhanceApp.js

```diff
@@ -1,14 +1,15 @@
+import Redirect from 'vuepress-theme-cloudlinux/components/Redirect.vue';
+
 export default ({
   Vue, // the version of Vue being used in the VuePress app
   options, // the options for the root Vue instance
   router, // the router instance for the app
   siteData // site metadata
 }) => {
-  const redirectionMapping = siteData.themeConfig.redirectionMapping;
+  Vue.component('Redirect', Redirect);
+
   router.beforeEach((to, from, next) => {
```

Comment:
Why we can't remove router logic from here and use it in the redirection component?

Reply:
Done